src/test/java/de/tum/cit/aet/artemis/core/DatabaseQueryCountTest.java

```diff
@@ -77,7 +77,7 @@ void testGetAllCoursesForDashboardRealisticQueryCount() throws Exception {
             var userCourse = request.get("/api/courses/" + course.getId() + "/for-dashboard", HttpStatus.OK, Course.class);
             log.info("Finish courses for dashboard call for one course");
             return userCourse;
-        }).hasBeenCalledTimes(12); // TODO: reduce this number back to 11
+        }).hasBeenCalledTimes(13); // TODO: reduce this number back to 11
```

Comment:
Why is it necessary to increase it even more, when the TODO states it should be reduced back to 11? Maybe update the TODO if it no longer makes sense

Reply:
i had to add an additional call for the FAQs of the course, thats why it increased. I dont know why this comment is still there, ill remove it